app/services/base_service.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+ 
+# Base class for service classes
+class BaseService 
+    private_class_method :new
```

Comment:
extra spaces

Reply:
the indentation is still wrong (4 spaces instead of 2). doesn't rubocop compain?